charts/leonardo/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: leonardo
-version: 0.0.3-RC1
+version: 09-15-20
```

Comment:
@choover-broad is this version format okay?

Reply:
nope, helm charts need to be semver!

Reply:
can split the existing version x.y.z and set it to x.(y+1).0